app/views/reset_password/edit.html.haml

```diff
@@ -0,0 +1,24 @@
+#header
+  .container
+    .row
+      .col-lg-4.col-lg-offset-4.col-md-6.col-md-offset-3.col-sm-8.col-sm-offset-2
+        %h1
+          %a{:href => root_path()} Epic Logger
+        .panel.panel-default
+          %h3 Reset password
+          = simple_form_for User.new, url: edit_reset_password_url(id: params[:id]), method: :get, wrapper: :vertical_form do |f|
```

Comment:
you can also write `simple_form_for :user` instead of `User.new`, tell me if that works!